common/protob/messages-nostr.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto2";
+package hw.trezor.messages.nostr;
+
+// Sugar for easier handling in Java
+option java_package = "com.satoshilabs.trezor.lib.protobuf";
+option java_outer_classname = "TrezorMessageNostr";
+
+import "options.proto";
+
+/**
+ * Request: Ask the device for the Nostr public key
+ * @start
+ * @next NostrMessageSignature
+ */
+message NostrGetPubkey {
+    repeated uint32 address_n = 1; // used to derive the key
+}
+
+/**
+ * Response: Nostr pubkey
+ * @end
+ */
+message NostrPubkey {
+    required bytes pubkey = 1;      // pubkey derived from the seed
+}
+
+/**
+ * @embed
+ */
+message NostrTag {
+    // Nostr tags consist of at least one string (the key)
+    // followed by an arbitrary number of strings,
+    // the first of which (if present) is called "value".
+    // See: NIP-01 https://github.com/nostr-protocol/nips/blob/master/01.md#tags
+    required string key = 1;
+    optional string value = 2;
+    repeated string extra = 3;
+}
+
+/**
+ * Request: Ask device to sign event
+ * @start
+ * @next NostrEventSignature
+ * @next Failure
+ */
+message NostrSignEvent {
+    repeated uint32 address_n = 1; // used to derive the key
+    required uint32 created_at = 2;
```

Comment:
I know it's a part of the protocol but add the comments what it means, e.g. `// UNIX timestamp set by the event creator`

Reply:
https://github.com/trezor/trezor-firmware/pull/4436/commits/126dc0b4a82bba4c131cc8f694a0eeece51d5f47#diff-a6b60857cdd027137830112c32a998813fd36181b11290bfa83ec573533e13f6R47